wf/__init__.py

```diff
@@ -0,0 +1,349 @@
+                    },
+                )
+                raise RuntimeError("The GPU ran out of memory.")
+            if "MMseqs2 API is giving errors" in realtime_output:
+                message(
+                    "error",
+                    {
+                        "title": f"MMseqs2 API Error",
+                        "body": "The MMseqs2 API is giving errors. Please contact support@latch.bio or message us via the chat in the bottom of the left sidebar.",
+                    },
+                )
+                raise RuntimeError("The MMseqs2 API is giving errors.")
+            print(realtime_output.strip(), flush=True)
+
+    retval = process.poll()
+    if retval != 0:
+        raise Exception(f"colabfold_batch failed with exit code {retval}")
+
+    if output_dir is not None:
+        latch_out_location = _fmt_dir(output_dir.remote_source) + f"/{run_name}"
+    else:
+        latch_out_location = f"latch:///ColabFold Outputs/{run_name}"
+
+    return LatchDir(
+        str((local_output).resolve()),
+        remote_directory=latch_out_location,
+    )
+
+
+@workflow
+def colabfold_mmseqs2_wf(
+    input_sequence_fork: str = "text",
+    output_location_fork: str = "default",
+    fasta_file: Optional[
+        Annotated[
+            LatchFile,
+            FlyteAnnotation(
+                {
+                    "rules": [
+                        {
+                            "regex": "(.fasta|.fa|.faa|.fas)$",
+                            "message": "Only .fasta, .fa, .fas, or .faa extensions are valid",
+                        }
+                    ],
+                }
+            ),
+        ]
+    ] = None,
+    aa_sequence: Optional[
+        Annotated[
+            str,
+            FlyteAnnotation(
+                {
+                    "appearance": {
+                        "type": "paragraph",
+                        "placeholder": ">SequenceOne\nLESPNCDWKNNR:RLENKNNCSPDW:CDWKNNENPDEA",
+                    },
+                    "rules": [
+                        {
+                            "regex": "^(>[^\n]+\n[A-Z]+(:[A-Z]+)*)(\n>.+\n[A-Z]+(:[A-Z]+)*)*$",
+                            "message": "Must be a list of named amino acid sequences, each formatted over two lines. The name line must start with `>` and the sequence line can only contain capital letters. For folding multimers, separate sequences under a single header with colons",
+                        }
+                    ],
+                }
+            ),
+        ]
+    ] = None,
+    custom_output_dir: Optional[LatchDir] = None,
+    nrof_models: NROFModels = NROFModels.one,
+    run_name: str = "run1"
+) -> LatchDir:
+    """The ColabFold version of AlphaFold2 is optimized for extremely fast predictions on small proteins. It uses the same basic architecture as AlphaFold2, but optimizes the sequence search procedure.
```

Comment:
can you use the new metadata object

Reply:
+1 can you also add tags so it renders nicely in all workflows